src/main/resources/db/migration/postgresql/R__6225_active_case_view.sql

```diff
@@ -4,7 +4,7 @@ DROP VIEW IF EXISTS active_case;
 
 CREATE VIEW active_case
```

Comment:
```suggestion
CREATE OR REPLACE VIEW active_case
```
We should be consistent with how we update the views. Remove the `DROP VIEW IF EXISTS active_case;` as well.